### PR TITLE
Implement L2 Token Contracts for Optimism

### PR DESCRIPTION
**Pull request by GoodluckH**
A new deployment script is added to support L2 token creation on Optimism networks. 

Added several fields to `.env.example` for L2 token deployment related secrets/variables.

Run `npm run deployL2TokenOptimismKovan` to test out L2 token contract creation.

Also added a note to `README` to address an build cache related issue when verifying contracts.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,3 +9,7 @@ DEFENDER_TEAM_API_SECRET_KEY=<SECRET_KEY>
 ARROW_TOKEN_PROXY_CONTRACT=<TOKEN_ADDRESS>
 ARROW_TOKEN_MUTISIG=<MULTISIG_ADDRESS>
 
+# L2 token deployment required secrets
+L1_TOKEN_ADDRESS=
+L2_TOKEN_NAME=Arrow Air
+L2_TOKEN_SYMBOL=ARROW
```
Comment (BrassLion) [inline .env.example]: Let's hardcode these into the deployment script. There's no way they'd change compared to L1.

File: README.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ npx hardhat deploy --network rinkeby --tags Deployment
 
 Deploying to mainnet is possible by subsituting all `--network rinkeby` arguments with `--network mainnet`.
 
+
+*Note: You may need to run `npx hardhat clean` to clear all the build caches and artifacts if you encounter "no such file or directory" issues when verifying contracts (also see [here](https://github.com/NomicFoundation/hardhat/issues/1117)).*
```
Comment (BrassLion) [inline README.md:62]: Did you find this solution after a pretty quick google? If so, we can probably remove this from the codebase, or move it to an issue if we really want to keep track of it.

File: deploy/deployArrowTokenL2.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "Creating instance of L2StandardERC20 on",
+    hre.network.name,
+    "network"
+  );
+  // Instantiate the Standard token factory
+  const l2StandardTokenFactory = new ethers.Contract(
+    "0x4200000000000000000000000000000000000012",
+    L2StandardTokenFactoryArtifact.abi,
+    signer
+  );
```
Comment (BrassLion) [inline deploy/deployArrowTokenL2.js:29]: This can only be run on Optimism, so a check that we are running on the right chain would be useful here.

**Review by BrassLion — CHANGES_REQUESTED** [review]

**Review by BrassLion — APPROVED** [review]
LGTM

**Comment by BrassLion** [comment]
It would be great if we could get the tests passing.

**Comment by GoodluckH** [comment]
Thanks. Resolved all the conflicts in `package-lock.json` that's causing the test CLI to fail.

**Pull request merged**